### PR TITLE
add hasura cloud integration

### PR DESCRIPTION
**Pull request by ShraddhaAg**
### What does this PR do?

This PR adds Hasura Cloud integration.

### CODE REVIEW

File: hasura_cloud/README.md
Comment (ChristineTChen) [inline hasura_cloud/README.md:8]:
```suggestion
The Datadog integration is an observability feature of Hasura Cloud that exports
```

File: hasura_cloud/assets/dashboards/hasura_cloud.json
Comment (ChristineTChen) [inline hasura_cloud/assets/dashboards/hasura_cloud.json:2]:
```suggestion
    "title": "Hasura Cloud Overview",
```

File: hasura_cloud/manifest.json
Comment (ChristineTChen) [inline hasura_cloud/manifest.json:23]:
```suggestion
    "monitoring", 
```

**Review by ChristineTChen — DISMISSED** [review]
Hey @ShraddhaAg , please add the integration and maintainer contact info to the CODEOWNERS file https://github.com/DataDog/integrations-extras/blob/master/.github/CODEOWNERS#L30-L31

File: hasura_cloud/README.md
Comment (cswatt) [inline hasura_cloud/README.md:24]:
```suggestion
Read the [Datadog Log Facets documentation][4] for information regarding creating facets from logs. 
```

File: hasura_cloud/README.md
Comment (cswatt) [inline hasura_cloud/README.md:34]:
```suggestion
The Hasura Cloud integration does not include any service checks.
```

File: hasura_cloud/README.md
Comment (cswatt) [inline hasura_cloud/README.md:38]:
```suggestion
The Hasura Cloud integration does not include any events.
```

File: hasura_cloud/README.md
Comment (cswatt) [inline hasura_cloud/README.md:42]:
```suggestion
Need help? Contact [Datadog support][6].
```
since the Hasura docs are also linked to, if a customer decides they need help from Hasura's side, they should know to contact Hasura support. our docs should direct customers to Datadog support for issues a customer feels to be on Datadog's side. support can then triage.

**Review by cswatt — CHANGES_REQUESTED** [review]
styleguide feedback

File: .github/CODEOWNERS
Comment (hithwen) [inline .github/CODEOWNERS:31]: There needs to be a github username or team handle before an email here. Validation is failing with the following error:
`Integration hasura_cloud has a `CODEOWNERS` entry, but the codeowner is not a username or team.`

**Review by hithwen — CHANGES_REQUESTED** [review]
Ensure all builds are green

**Review by coignetp — APPROVED** [review]
Hi, an internal request was made to display the logo on the integration tile, it can be merged after that

**Review by cswatt — APPROVED** [review]

**Pull request merged**
